evennia/web/webclient/static/webclient/js/evennia.js

```diff
@@ -50,6 +50,7 @@ An "emitter" object must have a function
 */
 
 (function() {
+    "use strict"
```

Comment:
The reason for not doing it at the file level is mostly to do with using script minifiers; if you ever use a minifier that concatenates all the script into a single script, then you're forced to run all of that script in strict mode. This would be fine for us currently, but if you ever take a dependency on a script that doesn't run in strict mode you wouldn't be able to minify it together with the rest of your scripts.

In any case, it's a best practice to enclose JS in a self-executing function to avoid leaking globals.